go-bindata-assetfs/main.go

```diff
@@ -118,6 +129,17 @@ func assetFS() *assetfs.AssetFS {
 func AssetFS() *assetfs.AssetFS {
 	return assetFS()
 }`)
+		fmt.Fprintln(out, `
+// AssetDIR similiar to assetFS() but requires a directory name
+func AssetDIR(dir string) *assetfs.AssetFS {
```

Comment:
Bindata already generates a function with name of `AssetDir` maybe `AssetDIR` is too similiar?